os/include/tinyara/timer.h

```diff
@@ -129,6 +130,19 @@
  * Public Types
  ****************************************************************************/
 
+enum timer_mode_e {
```

Comment:
Could you add some explanation about enum structures?

Reply:
yes, i added.